garrysmod/gamemodes/sandbox/gamemode/spawnmenu/controlpanel.lua

```diff
@@ -46,9 +46,8 @@ function PANEL:MatSelect( strConVar, tblOptions, bAutoStretch, iWidth, iHeight )
 
 	if ( tblOptions != nil ) then
 		for k, v in pairs( tblOptions ) do
-			local label = k
-			if ( isnumber( label ) ) then label = v end
-			MatSelect:AddMaterial( label, v )
+			local nam = isnumber( k ) and v or k
+			MatSelect:AddMaterial( nam, v )

```

Comment:
You really need to learn to stop including unrelated commits to a Pull Request. I will let it slide this time.